ui/app/components/app/transaction-list-item/index.scss

```diff
@@ -14,22 +14,24 @@
     padding: 16px 20px;
     display: grid;
     grid-template-columns: 45px 1fr 1fr 1fr;
-    grid-template-areas:
-      "identicon action status primary-amount"
-      "identicon nonce status secondary-amount";
+    grid-template-areas: "identicon action-and-nonce status amounts";
 
     @media screen and (max-width: $break-small) {
       padding: .5rem 1rem;
-      grid-template-columns: 45px 5fr 3fr;
-      grid-template-areas:
-        "nonce nonce nonce"
-        "identicon action primary-amount"
-        "identicon status secondary-amount";
+      grid-template-columns: 45px 3fr 3fr;
+      grid-template-areas: "identicon action-and-nonce status amounts";
     }
 
     &:hover {
       background: rgba($alto, .2);
     }
+
+    .transaction-status {
```

Comment:
This seems like it would be more appropriate to add to `ui/app/components/app/transition-status/index.scss`.

In fact, `min-width: 72px` is already set there. It's just missing the width.